plugins/scripts/create_desktop_file.py

```diff
@@ -48,7 +48,6 @@ def create_desktop_file(
 Type=Application
 Categories=X-Polaris
 Icon={icon}"""
-
```

Comment:
if this was auto format yea, leave a line of visual separation between the strings, rather remove one line before the shortcut parsing, line 43 ...

Reply:
ah yes, added this back. I originally was checking to see if version was being passed in and was going to adjust the desktop file naming convention, but then shifted and re-purposed it for Juno nuke.

updated.